crates/dprint/src/utils/glob/glob_matcher.rs

```diff
@@ -151,6 +158,20 @@ mod test {
     assert!(!glob_matcher.is_match("/testing/dir/no-match.ts"));
   }
 
+  #[test]
+  fn handles_external_files() {
+    let cwd = CanonicalizedPathBuf::new_for_testing("/testing/dir");
+    let glob_matcher = GlobMatcher::new(
+      GlobPatterns {
+        includes: vec![GlobPattern::new("/testing/dir/*.ts".to_string(), cwd.clone())],
+        excludes: vec![GlobPattern::new("/testing/dir/no-match.ts".to_string(), cwd.clone())],
+      },
+      &GlobMatcherOptions { case_sensitive: true },
+    )
+    .unwrap();
+    assert!(!glob_matcher.is_match("/some/other/dir/file.ts"));
+  }
```

Comment:
This test will result in a panic without the code above as `strip_prefix` will error.